server/src/main/resources/crafter/profile/init-data.json

```diff
@@ -0,0 +1,146 @@
+        {
+          "allowedActions" : [ "*" ],
+          "tenant" : "*"
+        }
+      ]
+    },
+    {
+      "_id" : "b4d44030-d0af-11e3-9c1a-0800200c9a66",
+      "application" : "engine",
+      "master": false,
+      "tenantPermissions" : [
+        {
+          "allowedActions" : [ "READ_TENANT", "MANAGE_PROFILES", "MANAGE_TICKETS" ],
+          "tenant" : "*"
+        }
+      ]
+    },
+    {
+      "_id" : "2ba3ac10-c43e-11e3-9c1a-0800200c9a66",
+      "application" : "social",
+      "master": false,
+      "tenantPermissions" : [
+        {
+          "allowedActions" : [ "READ_TENANT", "MANAGE_PROFILES", "MANAGE_TICKETS" ],
+          "tenant" : "*"
+        }
+      ]
+    }
+  ],
+  "tenant": [{
+    "name" : "default",
+    "verifyNewProfiles" : false,
+    "availableRoles" : [ "PROFILE_SUPERADMIN", "PROFILE_TENANT_ADMIN", "PROFILE_ADMIN", "SOCIAL_SUPERADMIN" ],
+    "attributeDefinitions" : [
+      {
+        "name" : "firstName",
+        "metadata": {
+          "label": "First Name",
+          "type": "TEXT",
+          "displayOrder": 0
+        },
+        "permissions" : [
+          {
+            "application" : "*",
+            "allowedActions" : [ "*" ]
+          }
+        ]
+      },
+      {
+        "name" : "lastName",
+        "metadata": {
+          "label": "Last Name",
+          "type": "TEXT",
+          "displayOrder": 1
+        },
+        "permissions" : [
+          {
+            "application" : "*",
+            "allowedActions" : [ "*" ]
+          }
+        ]
+      },
+      {
+        "name" : "displayName",
+        "metadata": {
+          "label": "Display Name",
+          "type": "TEXT",
+          "displayOrder": 2
+        },
+        "permissions" : [
+          {
+            "application" : "*",
+            "allowedActions" : [ "*" ]
+          }
+        ]
+      },
+      {
+        "name" : "avatarLink",
+        "metadata": {
+          "label": "Avatar Link",
+          "type": "TEXT",
+          "displayOrder": 3
+        },
+        "permissions" : [
+          {
+            "application" : "*",
+            "allowedActions" : [ "*" ]
+          }
+        ]
+      },
+      {
+        "name": "socialContexts",
+        "metadata": {
+          "label": "Social Contexts",
+          "type": "COMPLEX",
+          "displayOrder": 4
+        },
+        "permissions": [
+          {
+            "application": "*",
+            "allowedActions": [ "*" ]
+          }
+        ]
+      },
+      {
+        "name": "connections",
+        "metadata": {
+          "label": "Connections",
+          "type": "COMPLEX",
+          "displayOrder": 5
+        },
+        "permissions": [
+          {
+            "application": "*",
+            "allowedActions": [ "*" ]
+          }
+        ]
+      }
+    ]
+  }],
+  "profile": [{
+    "username" : "admin",
+    "password" : "4rQ8a67wAk1GRwIqHix5kYw1MORa49o83Y7zXQhBqT0=|j4vsWtPbYjO3LfSiQcnGlw==",
```

Comment:
What's this?

Reply:
@sumerjabri it's the default `admin/admin` username/password. The `init-data.json` is the migrated version from `init-data.js`.
Instead of using `mongosh` to directly run `init-data.js`, we created a JSON version then read the file and insert with Java Client